test/specs/utils/AutoControlledComponent-test.js

```diff
@@ -39,6 +39,8 @@ describe('extending AutoControlledComponent', () => {
     })
 
     it('sets state for autoControlledProps', () => {
+      consoleUtil.disableOnce()
+
```

Comment:
These test throw auto controlled prop warnings due to randomly generated props.  Silenced logs to silence the warnings.